jcl/source/common/JclExprEval.pas

```diff
@@ -1194,6 +1195,11 @@ function TExprCompileParser.CompileExprLevel2(ASkip: Boolean): TExprNode;
         Result := NodeFactory.Multiply(Result, CompileExprLevel3(True));
       etForwardSlash:
         Result := NodeFactory.Divide(Result, CompileExprLevel3(True));
+      etPercent:
+        begin
+          Result := NodeFactory.Divide(Result, NodeFactory.LoadConst(100));
+          Lexer.NextTok;
```

Comment:
Why is the NextTok call required here? It's not used in any of the other elements of the case which makes me wonder...